examples/src/main/java/org/apache/spark/examples/ml/JavaCountVectorizerExample.java

```diff
@@ -54,6 +54,7 @@ public static void main(String[] args) {
       .setOutputCol("feature")
       .setVocabSize(3)
       .setMinDF(2)
+      .setBinary(true)
```

Comment:
I'm neutral on it - my point is that the examples generally show "normal" (or "default") usage, and are intended to be short, succinct illustrations of usage. It's not necessary to show every possible param that can be set in each example.

In this particular case, I would actually say that `binary=true` is a slightly "unusual" use case (not completely expert but certainly not the "normal" use case), so let's remove it from this PR.

For `relativeError`, it's not "expert" (though there's an argument that it could be made an `expertParam`), but in almost all cases it is not necessary to deviate from the default, so again this can be removed.